### PR TITLE
chore(README): update readme

### PR DESCRIPTION
**Pull request by whalecold**
**What this PR does / why we need it**:

chore(README): update readme

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by whalecold** [comment]
/cherrypick release-1.4
@zxq-bit @hezhizhen

File: README.md [hunk 1]
```diff
@@ -57,6 +57,7 @@ _config:
     template:             # Chart 模板信息，为模板自动升级提供信息
       type: string        # Chart 模板类型
       version: semvar     # Chart 模板版本号
+    appVersion: string    # Chart 创建 workload 时的版本号，会添加到 pod 的 label 中，目前多用与灰度发布。
```
Comment (hezhizhen) [inline README.md]: 于

Comment (whalecold) [reply README.md]: done

**Pull request merged**